src/creature_states_gardn.c

```diff
@@ -186,7 +186,7 @@ void person_search_for_food_again(struct Thing *creatng, struct Room *room)
     } else
     {
         near_food_tng->byte_15 = 255;
-        near_food_tng->byte_16 = 127;
+        near_food_tng->hit_type = 127;
```

Comment:
It is unrelated to hit_type. It is just lives at same relative location. Please replace it with `near_food_tng->food.byte_16 = 127;`

Reply:
I'll make this change, but I find it weird because I kept all the food.byte_16 original. Is food.byte_16 somehow the same to regular byte_16? This one was just byte_16 before the change.

Reply:
food.byte_16 is exactly same as byte_16. Pure logical change. Here we know that it is a food and someday we will name it like `food.remain_ticks` or `food.something_else` but it will still reside at relative offset of 16 bytes